### PR TITLE
[mobile] Apply Winner screen width to the dialog

### PR DESCRIPTION
**Pull request by DohyungLim**
- Add Winner screen width.
- Remove unnecessary list style to dialog less.

### CODE REVIEW

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -11,16 +11,25 @@
 	background-color: @color_popup_bg;
 	border-radius: 26 * @px_base;;
 
-	@media (orientation: portrait) and (max-width: 479px) {
+	// mobile
+	@media (min-width: 0px) and (max-width: 479px) {
 		width: 100%;
 	}
-	@media (orientation: landscape) and (max-height: 479px) {
+	@media (min-height: 0px) and (max-height: 479px) {
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: Do you need to check the max-height?

File: src/css/profile/mobile/common/popup.less [hunk 2]
```diff
@@ -11,16 +11,25 @@
...
-	@media (orientation: portrait) and (min-width: 673px) {
+	//winner
+	@media (min-width: 480px) and (max-width: 672px) and (min-height: 480px) {
+		width: 63%;
+	}
+	@media (min-height: 480px) and (max-height: 672px) and (min-width: 480px) {
+		width: 55%;
+	}
+	//tablet
+	@media (min-width: 673px) and (max-width: 985px) and (min-width: 673px) {
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: Why did you separate 
	@media (min-height: 480px) and (max-height: 672px) and (min-width: 480px) {
		width: 55%;
	}

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/popup.less]: I think so, you don't need to check the min-height.

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: yes. I agree comment.
I will remove it.

done.

File: src/css/profile/mobile/common/popup.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Review by singa2000 — APPROVED** [review]
It is implemented as Guide. Good.

**Pull request merged**